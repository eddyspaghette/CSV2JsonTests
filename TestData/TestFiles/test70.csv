ッ;あ,が,ざ,℁,⏳
ヅ|い,け,₠,Å,○
ミ,,げ|₡,ℬ,☝
デ,う,こ,₢,Δ,⭐
ト,ぇ,ご,₣,ψ,😵
ド,え,さ,℀,,🍕
ツ,ぃ,ぎ,せ,℟,▓
