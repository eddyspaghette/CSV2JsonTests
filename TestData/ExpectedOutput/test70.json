[
 {
   "ッ;あ": "ヅ|い",
   "が": "け",
   "ざ": "₠",
   "℁": "Å",
   "⏳": "○",
   "FIELD6": ""
 },
 {
   "ッ;あ": "ミ",
   "が": "",
   "ざ": "げ|₡",
   "℁": "ℬ",
   "⏳": "☝",
   "FIELD6": ""
 },
 {
   "ッ;あ": "デ",
   "が": "う",
   "ざ": "こ",
   "℁": "₢",
   "⏳": "Δ",
   "FIELD6": "⭐"
 },
 {
   "ッ;あ": "ト",
   "が": "ぇ",
   "ざ": "ご",
   "℁": "₣",
   "⏳": "ψ",
   "FIELD6": "😵"
 },
 {
   "ッ;あ": "ド",
   "が": "え",
   "ざ": "さ",
   "℁": "℀",
   "⏳": "",
   "FIELD6": "🍕"
 },
 {
   "ッ;あ": "ツ",
   "が": "ぃ",
   "ざ": "ぎ",
   "℁": "せ",
   "⏳": "℟",
   "FIELD6": "▓"
 }
]